package.json

```diff
@@ -50,15 +50,20 @@
     "split2": "^2.1.1",
     "std-mocks": "^1.0.0",
     "through2": "^2.0.0",
-    "winston-compat": "^0.1.1"
+    "winston-compat": "^0.1.1",
+    "babel-cli": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "rimraf": "^2.6.2"
   },
-  "main": "./lib/winston",
+  "main": "./build/winston",
```

Comment:
Why not use `"browser"` for this?

Reply:
To clarify, do you mean why not name the directory "browser" instead of "build"? We could call it whatever you'd like, but it will depend on your strategy for the module (this will relate to your request to remove "lib" from `.npmignore`). By publishing the "lib" directory _and_ the "build" directory, I'm assuming that you'd like to publish two different versions of the source code: one that is not pre-compiled and won't work in a browser (the "lib" folder) and one that is pre-compiled and will work in either a `node` process _and_ a browser (the "build" folder).

The way I structured it runs under the assumption that you only want to ship the pre-compiled code. This approach has the advantage that the code will work for either a browser or a node process, but it will eliminate the "lib" folder from the published code, which will break anyone that's referencing files directly from the module. Publishing both pre-compiled and raw code will work as well, but it will increase the size of the module and will require anyone wanting to use the module in a browser to explicitly include the "build" path. For example, `require('winston')` would not work in a browser, but `require('winston/build')` would.